sdk/keyvault/keyvault-admin/src/accessControlClient.ts

```diff
@@ -95,26 +95,38 @@ export class KeyVaultAccessControlClient {
       ? challengeBasedAuthenticationPolicy(credential)
       : signingPolicy(credential);
 
-    const internalPipelineOptions = {
-      ...pipelineOptions,
-      ...{
-        loggingOptions: {
-          logger: logger.info,
-          logPolicyOptions: {
```

Comment:
This wasn't working! We had a bug here.

Reply:
Would TS have caught the problem if we had spread `pipelineOptions` but not the inline loggingOptions?

Reply:
I'll bring the spread operator back here!